Prescription.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+
+namespace задание11
+{
+    public enum Prescription
+    {
+        underRecipe,
+        withoutRecipe

```

Comment:
Лучше с большой буквы
```suggestion
        UnderRecipe,
        WithoutRecipe

```